hr_employee_social_media/models/hr_social_media.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# copyright  2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrSocialMedia(models.Model):
+    _name = 'hr.social.media'
+
+    name = fields.Char(
+        string='Name',
+    )
+    website = fields.Char(
```

Comment:
pls, do not use "website" as field name. It can collide many times. Be more explicit and use something like "social_url".